samples/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx

```diff
@@ -71,8 +71,8 @@ export default class CustomTimeRangeSelector extends React.Component {
                         <MenuItem value={'day'}>Day</MenuItem>
                         <MenuItem value={'hour'}>Hour</MenuItem>
                         <MenuItem value={'minute'}>Minute</MenuItem>
-                        <MenuItem value={'seconds'}>Seconds</MenuItem>
-                        <MenuItem value={'milliseconds'}>Milliseconds</MenuItem>
```

Comment:
can we actually show Milliseconds ?

Reply:
I have created an [issue](https://github.com/wso2/carbon-dashboards/issues/988) regarding this. Please follow up.